src/Stratis.Bitcoin.Features.Wallet.Tests/WalletControllerTest.cs

```diff
@@ -1267,6 +1267,66 @@ public void BuildTransactionWithValidRequestAllowingUnconfirmedReturnsWalletBuil
             Assert.Equal(sentTrx.GetHash(), model.TransactionId);
         }
 
+        [Fact]
+        public void BuildTransactionWithCustomFeeAmountAndFeeTypeReturnsWalletBuildTransactionModelWithFeeAmount()
```

Comment:
Can you add underscores in the method name, its not easy to read

Reply:
I'm going to leave it like this for now as the other tests in the class don't have underscores, but we'll change when we revisit naming of tests.